backend/src/modules/tournament-tree/core/domain/TournamentTree.ts

```diff
@@ -5,19 +5,21 @@ import { EntityIdGenerator } from '../../../../shared/core/application/EntityIdG
 import { DomainCommandResult } from '../../../../shared/core/domain/DomainCommandResult';
 import { CurrentTimeProvider } from '../../../../shared/core/CurrentTimeProvider';
 import { TournamentTreeWasCreated } from './event/TournamentTreeWasCreated';
+import { TournamentTeamId } from './TournamentTeamId';
 
 export class TournamentTree {
   readonly tournamentTeams: TournamentTeam[];
   readonly tournamentTreeArray: FightingTeamsGroup[];
   readonly tournamentId: string;
+  firstPlaceTeamId: string | undefined = undefined;
 
   private constructor(props: { tournamentId: string; tournamentTreeArray: FightingTeamsGroup[]; tournamentTeams: TournamentTeam[] }) {
     this.tournamentTeams = props.tournamentTeams;
     this.tournamentTreeArray = props.tournamentTreeArray;
     this.tournamentId = props.tournamentId;
   }
 
-  static createSingleTournamentTree(props: {
+  public static createSingleTournamentTree(props: {
```

Comment:
czy ta metoda musi być `public` ? w sensie jeśli jest to zamierzone to spoko, a jeśli do testów to niekoniecznie :P

Reply:
chociaż w sumie to może niepotrzebnie się czepiam xD fajnie, że ten algorytm jest przetestowany

Reply:
Ha, dobry komentarz. Aż sam musiałem sprawdzić co stanie się jak dam private.
Wcześniej ona i tak była publiczna, teraz dodałem tylko to słowo, ale jak je usunę dalej będzie public.
Korzysta z niej funkcja zewnętrzna która po wywołaniu komendy/zapytania, wywołuje ją.

Reply:
wybacz Piter, oczywiście, że defaultowo są publiczne, chyba się dzisiaj nie wyspałem :P